docusaurus/docs/how-to-guides/ui-extensions/expose-a-component.md

```diff
@@ -11,13 +11,15 @@ keywords:
 sidebar_position: 30
 ---
 
-Exposing components allows for App plugins to easily share functionality with other App plugins. Compared to [registering an extension](link), they do not require the component extension provider to explicitly register against any extension points, and can therefore be [used by any App plugin](link-to-use) with no action required by the provider.
+Expose components to allow app plugins to easily share functionality with other app plugins. Compared to [registering an extension](./register-an-extension), they do not require the extension provider to explicitly register a component against any extension points, and can therefore be [used by any app plugin](link-to-use) with no action required by the provider.
```

Comment:
I'm not sure this reads right to me anymore. 

If we can't say exposing. We could instead say:
> Expose components from your app plugin so they can be imported by any other app plugin, as a way of easily sharing functionality. Compared to [registering an extension](./register-an-extension), they do not require the extension provider to explicitly register the component against any extensions points, and can therefore be [used by any app plugin](./use-an-exposed-component) with no action required by the provider.

Note change from `a component` to `the component` and added link for used by any app plugin.

Reply:
made suggestion to fix